api/students.jsx

```diff
@@ -0,0 +1,14 @@
+import axios from "axios";
+
+const BASE_URL =
+  "https://677278dcee76b92dd492659a.mockapi.io/students/students";
+
+export const createStudent = async (studentData) => {
+  try {
+    const response = await axios.post(BASE_URL, studentData);
```

Comment:
For short projects yes this is good, but large projects we shouldn't use base_url to every request. we need to add some default config to axios and add some interceptors.

 please go through the docs 
 
[ Default config](https://axios-http.com/docs/config_defaults)
[Interceptors](https://axios-http.com/docs/interceptors)